nextflow_schema.json

```diff
@@ -85,6 +85,17 @@
                     "description": "Path to GFF3 annotation file.",
                     "help_text": "This parameter must be specified if `--genome` or `--gtf` are not specified."
                 },
+                "skip_gtf_filter": {
```

Comment:
I wonder whether that should rather be inside the `process_skipping_options` section of the parameters?

But it is also reasonable to have at this spot, so rather a question than a fixed opinion.

Reply:
Thanks! I'll let @drpatelh make that call

Reply:
Good point 👍🏽 I'll move it.

Reply:
Fixed https://github.com/nf-core/rnaseq/pull/1107/commits/e7126c80966e6d2999cc5db3f9b5bbe0e98a9ba5